packit/utils/changelog_helper.py

```diff
@@ -70,7 +70,8 @@ def get_entry_from_action(
             Changelog entry or `None` if action is not present.
         """
         resolved_bugs_str = " ".join(resolved_bugs) if resolved_bugs else ""
-        env = {
+        package_config = self.up.package_config if self.up else self.dg.package_config
```

Comment:
If I understand correctly, this will use the upstream config even when running `pull-from-upstream`, won't it?

Reply:
Yes, I expect that `pull-from-upstream` and `propose-downstream` have both the `self.up`, other jobs should have `self.dg`. But I didn't try it out, so I may be wrong.

Reply:
We have upstream in `pull-from-upstream` too, and there (usually) isn't config… won't this fail? :eyes: sorry for the nagging, but now I realize that it seems like an easy fix, but in fact is more complicated :D